tasks/main-clevis.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Ensure required packages are installed
+  package:
+    name: "{{ __nbde_client_package_list }}"
+    state: present
+
+
+- name: Prepare keyfiles, perform clevis operations and dispose of keyfiles
+  block:
+    - name: Ensure we transfer keyfiles
+      copy:
+        src: "{{ item.keyfile | default('/dev/null') }}"
+        dest: "{{ __nbde_client_managed_dir }}/"
+        mode: '0400'
+      loop: "{{ nbde_client_bindings }}"
+
+    - name: Perform clevis operations
+      nbde_client_clevis:
+        bindings: "{{ nbde_client_bindings | default([]) }}"
```

Comment:
Does the module actually do anything if `nbde_client_bindings` is empty?  Should we just skip the module in that case? e.g.  add a `when: nbde_client_bindings | d([])` to this task

Reply:
Nope, it does nothing and can be skipped in that case.

Reply:
It's being skipped now, thanks.